Dockerfile.assisted-service

```diff
@@ -35,7 +35,33 @@ RUN cd build && python3 setup.py sdist --dist-dir /assisted-service-client
 FROM quay.io/centos/centos:centos8
 
 # openshift-install requires this
-RUN yum install -y libvirt-libs && yum clean all
+RUN dnf install -y libvirt-libs git python3 && \
+    dnf install -b -y  dnf-plugins-core && \
```

Comment:
this is how it's being installed for CNV?

https://github.com/nmstate/kubernetes-nmstate/blob/master/build/Dockerfile#L4

what am I missing?

Reply:
no, for cnv it is being installed from official rpm package, we don't have it yet